tests/clusters/test_cluster.py

```diff
@@ -79,6 +79,29 @@
     },
 ]
 
+PARTITIONED_CLUSTERS_CONFIG = [
+    {
+        "host": "host_slice",
+        "port": 9000,
+        "user": "default",
+        "password": "",
+        "database": "default",
+        "http_port": 8123,
+        "storage_sets": {("generic_metrics_distributions", 0)},
```

Comment:
nit: I'd call this `storage_set_partitions` or something

Reply:
+1 to this suggestion